bigtable/client/internal/table.cc

```diff
@@ -120,7 +123,16 @@ std::vector<FailedMutation> Table::BulkApply(BulkMutation&& mut,
 }
 
 RowReader Table::ReadRows(RowSet row_set, Filter filter, bool raise_on_error) {
-  return RowReader(client_, table_name(), std::move(row_set),
+  if (!app_profile_id_.get().empty()) {
```

Comment:
Is there a reason for this check?  I think if the profile id is empty the effect is the same?